site/content/integrate/apps/api/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "API"
+heading: "Mattermost Apps API Reference"
+description: "This reference describes the Mattermost Apps API"
+weight: 300
+---
+
+## App Structure
+
+Apps consist of 3 kinds of assets:
+- [manifest.json]({{< ref "manifest" >}}) file
+- [Call]({{< ref "call" >}}) handlers, starting with `/bindings` (see
+  [Interactivity]({{< ref "interactivity" >}})) and `/on_install` (see
+  [Manifest]({{< ref "manifest" >}})). Call handlers are typically source files
+  in a programming language like JavaScript, Python, or Go.
+- Static assets, currently solely icon files
+
+## Interactivty: Bindings and Forms
+
+- To interact with users an App must bind a call to a location in the Mattermost
+  user interface. These locations may be `/`-commands, toolbar and menu items,
+  and embedded in posts.
+- A binding may display a `[Form]({{< ref "interactivity" >}})`, or it may
```

Comment:
```suggestion
- A binding may display a [`Form`]({{< ref "interactivity" >}}), or it may
```

Reply:
removed the formatting, normal links are better